scripts/src/setup.ts

```diff
@@ -0,0 +1,207 @@
+  }
+  console.log(
+    `BTC: ${BTCTokenID}, ETH: ${ETHTokenID}, USDC: ${USDCTokenID}, USDT: ${USDTTokenID}, SOL: ${SOLTokenID}`
+  )
+
+  const response = await fetch(
+    'https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=bitcoin,ethereum,alephium,solana'
+  )
+  const data = await response.json()
+  const prices = {
+    [BTCTokenID]: data.find((coin: any) => coin.id === 'bitcoin').current_price,
+    [ETHTokenID]: data.find((coin: any) => coin.id === 'ethereum').current_price,
+    [USDCTokenID]: 1,
+    [USDTTokenID]: 1,
+    [SOLTokenID]: data.find((coin: any) => coin.id === 'solana').current_price,
+    [ALPH_TOKEN_ID]: data.find((coin: any) => coin.id === 'alephium').current_price
+  }
+  console.log(
+    `BTC: ${prices[BTCTokenID]}, ETH: ${prices[ETHTokenID]}, USDC: ${prices[USDCTokenID]}, USDT: ${prices[USDTTokenID]}, SOL: ${prices[SOLTokenID]}, ALPH: ${prices[ALPH_TOKEN_ID]}`
+  )
+
+  const poolKeys: [PoolKey, bigint][] = [
+    // [newPoolKey(ALPH_TOKEN_ID, BTCTokenID, FEE_TIERS[1]), 10804609546189987720n],
+    // [newPoolKey(ALPH_TOKEN_ID, ETHTokenID, FEE_TIERS[1]), 4711830510277394610468n],
+    // [newPoolKey(ALPH_TOKEN_ID, USDCTokenID, FEE_TIERS[1]), 272063075569508447756n],
+    // [newPoolKey(ALPH_TOKEN_ID, USDTTokenID, FEE_TIERS[1]), 272063075569508447756n],
+    // [newPoolKey(ALPH_TOKEN_ID, SOLTokenID, FEE_TIERS[1]), 37143700245489847211n],
+    [newPoolKey(BTCTokenID, ETHTokenID, FEE_TIERS[1]), 130559235944405760n],
+    [newPoolKey(BTCTokenID, USDCTokenID, FEE_TIERS[1]), 7865049221247086n],
+    [newPoolKey(BTCTokenID, USDTTokenID, FEE_TIERS[1]), 7865049221247086n],
+    [newPoolKey(BTCTokenID, SOLTokenID, FEE_TIERS[1]), 977937074251981n],
+    [newPoolKey(ETHTokenID, USDCTokenID, FEE_TIERS[1]), 3454809855596621497n],
+    [newPoolKey(ETHTokenID, USDTTokenID, FEE_TIERS[1]), 3454809855596621497n],
+    [newPoolKey(ETHTokenID, SOLTokenID, FEE_TIERS[1]), 423131631710393596n],
+    [newPoolKey(USDCTokenID, USDTTokenID, FEE_TIERS[1]), 9999818389598293n],
+    [newPoolKey(USDCTokenID, SOLTokenID, FEE_TIERS[1]), 24911294718392400n],
+    [newPoolKey(USDTTokenID, SOLTokenID, FEE_TIERS[1]), 24911294718392400n]
+  ]
+  for (const [poolKey] of poolKeys) {
+    const price =
+      (1 / (prices[poolKey.tokenY] / prices[poolKey.tokenX])) *
+      10 ** (Number(decimals[poolKey.tokenY]) - Number(decimals[poolKey.tokenX])) *
+      10 ** 24
+    try {
+      const poolSqrtPrice = priceToSqrtPrice(BigInt(Math.round(price)) as Price)
+      await invariant.createPool(
+        account,
+        poolKey.tokenX,
+        poolKey.tokenY,
+        poolKey.feeTier,
+        poolSqrtPrice
+      )
+    } catch (e) {
+      console.log('Create pool error', poolKey, e)
+    }
+  }
+  console.log('Successfully added pools')
+
+  const token = FungibleToken.load(network)
+  await token.mint(account, (2n ** 96n - 1n) as TokenAmount, BTCTokenID)
+  await token.mint(account, (2n ** 96n - 1n) as TokenAmount, ETHTokenID)
```

Comment:
I am not sure if it is needed, but on alephium we can mint more tokens because of `U256` type.

Reply:
I am aware, but faucets are filled up till 2**256-1 and that's probably where it matters